Syndication/AzureStack.MarketplaceSyndication.psm1

```diff
@@ -19,6 +19,9 @@ function Export-AzSOfflineMarketplaceItem {
         [ValidateNotNullorEmpty()]
         [String] $ResourceGroup = "azurestack",
 
+        [Parameter(Mandatory = $false, ParameterSetName = 'SyncOfflineAzsMarketplaceItem')]
+        [Switch] $ReduceDownloadThreads = $false,
```

Comment:
AzCopyDownloadThreads might be a better name

Reply:
fixed, thanks